proxy/src/main/java/com/velocitypowered/proxy/connection/client/ClientPlaySessionHandler.java

```diff
@@ -482,12 +482,13 @@ public CompletableFuture<Void> doSwitch() {
       // Send keep alive to try to avoid timeouts
       player.sendKeepAlive();
 
-      // Reset Tablist header and footer to prevent desync
-      player.clearHeaderAndFooter();
+      // Config state clears everything in the client. No need to clear later.
```

Comment:
just to check, this is not assumative towards 1.20.2 behavior? i.e. this isn't going to have a negagtive impact for older versions?

Reply:
Correct, doSwitch is only called for 1.20.2 players.

Reply:
![image](https://github.com/PaperMC/Velocity/assets/1228900/81ddcf9e-f86b-42cc-a8b6-3952dea51bf3)